client/overview/task-list/index.js

```diff
@@ -32,20 +175,37 @@ const TaskList = ( { tasks } ) => {
 			</CardHeader>
 			<CardBody>
 				<CollapsibleList
+					animation="slide-right"
 					collapsed={ false }
+					show={ 5 }
```

Comment:
I am good with this change, but I am just curious where this criteria was mentioned, I couldn't find the acceptance criteria for this anywhere.

Reply:
I was sure that I had read that but I couldn't find where so I guess it was my imagination 😅

Maybe @elizaan36 can confirm how many items should be shown in the `Things to do` list.

Reply:
On the home screen, there's two tasks shown by default. On the payments overview, 5 by default sounds good since there's a lot less content on the screen.